PencilCase/PencilCase/scripts/typings/knockout/knockout.d.ts

```diff
@@ -62,7 +62,8 @@ interface KnockoutSubscription {
 
 interface KnockoutSubscribable<T> extends KnockoutSubscribableFunctions<T> {
     subscribe(callback: (newValue: T) => void, target: any, event: "beforeChange"): KnockoutSubscription;
-	subscribe(callback: (newValue: T) => void, target?: any, event?: "change"): KnockoutSubscription;
+    subscribe(callback: (newValue: T) => void, target?: any, event?: "change"): KnockoutSubscription;
+    subscribeChanged(callback: (newValue: T, oldValue: T) => void, target?: any, dataContext?: any): KnockoutSubscription;
```

Comment:
Add this subscribeChanged definition into KO type definition file to make compiling pass.